getcoverage.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+CPP_ETHEREUM_PATH=$(pwd)
+
+if [ ! -d "$CPP_ETHEREUM_PATH/build/test" ]; then
+   echo "You need to compile and build ethereum with cmake -DPROFILING option to the build dir!"
+   exit;	 
+fi
+
+if which lcov >/dev/null; then
+    if which genhtml >/dev/null; then
+	echo Running testeth...
+	$($CPP_ETHEREUM_PATH/build/test/testeth)
+	echo Prepearing coverage info...
+    else
+	echo genhtml not found
+	exit;
+    fi
+else
+    echo lcov not found
+    exit;
+fi
+
+OUTPUT_DIR="$CPP_ETHEREUM_PATH/build/test/coverage"
+
+TESTETH=$CPP_ETHEREUM_PATH/build/test/CMakeFiles/testeth.dir
+lcov --capture --directory $TESTETH --output-file $OUTPUT_DIR/coverage.info
```

Comment:
After capturing the data you should extract only cpp-ethereum related data. `lcov --extract */cpp-ethereum/*`.